### ISSUE TITLE
@pc-vue3-ui这个是私有化的吗?

### ISSUE THREAD

**Issue by wenghaoping** [GGXXMM/vue3-ui#1]
"@pc-vue3-ui/components": "workspace:^",
    "@pc-vue3-ui/theme-chalk": "workspace:^",
这2个库是私有化的吗,发现没搜到.

**Comment by GGXXMM** [comment]
> "@pc-vue3-ui/components": "workspace:^", "@pc-vue3-ui/theme-chalk": "workspace:^", 这2个库是私有化的吗,发现没搜到.

是的。monorepo 方便管理本地依赖包，pnpm install的时候会识别 pnpm-workspace.yaml 配置的本地依赖。
```
# 进入项目根目录，安装本地依赖
pnpm install  @pc-vue3-ui/components @pc-vue3-ui/theme-chalk -w
```
